server/views/pages/continueReportSelect.njk

```diff
@@ -52,20 +48,19 @@
           {{ govukTable({
         head: [
           {
-            text: "Prisoner’s name",
+            text: "Name and prison number",
             attributes: {
               "aria-sort": "ascending",
               "data-qa": "prisoner-name-sort"
             }
           },
-          { text: "Date of incident", 
+          { text: "Discovery date and time", 
             attributes: {
               "aria-sort": "ascending",
```

Comment:
I think we need to check with Simon whether he wants this data sorting ascending or descending - it looks like in the screenshot he wants it descending (at the minute it's ascending)

Reply:
well spotted!